### DISCUSSION TITLE
Show which Firefox version Floorp is based on in the about dialog

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#1126] [Feature Request]
The only way to see it is in about:preferences#general and under "Updates", it would be quicker to see it under Floorp > About (on macOS). Perhaps it could be put next to the version number, before "what's new". Ex: `| Firefox 115` like about:preferences.

![CleanShot 2024-05-01 at 09 34 38@2x](https://github.com/Floorp-Projects/Floorp/assets/42646062/c9e66cfd-9093-4f06-abb6-72d9712241ee)

**Comment by rguhr** [comment]
Looks like this was implemented.

<img width="688" height="485" alt="image" src="https://github.com/user-attachments/assets/03abd18d-1b0f-4eab-bd37-8e299bbedae9" />

**Comment by CutterKnife** [comment]
Closing this as complete. Thank you!
